docs/src/howtouse/creatingModels.md

```diff
@@ -69,9 +69,9 @@ cell structure with auxiliar params information, required for some element types
  The `massMatType` field sets, for frame or truss elements, whether consistent or lumped mass matrix is used for the inertial term in dynamic analyses. The `massMatType` field should be set as a string variable: `'consistent'` or `'lumped'`,  and if it is not declared then by default the `'lumped'` mass matrix is set.
 
  ### `elements.elemTypeAero`
-The `elementTypeAero` field is a vector that sets for frame aerodynamic co-rotational element the chord vector in total-deformed coordinates $t$ (which initially are equal to reference $e$), and the number of gauss points $numGauss$:
+The `elementTypeAero` field is a vector that sets for frame aerodynamic co-rotational element the chord vector in total-deformed coordinates $t$ (which initially must be expressed in the coordinates of system $e$): Also the number of gauss points $numGauss$ and the $computeAeroBool$ (which computes the tangent matrix of the aerodynamic force vector) are set:
```

Comment:
```suggestion
The field  `elementTypeAero` should contain a vector with the three coordinates of the aerodynamic chord vector. The system of coordinates considered for this is the local reference system at the undeformed configuration. Also the number of Gauss integration points $numGauss$ and the $computeAeroBool$ (which computes the tangent matrix of the aerodynamic force vector) are set:
```

Reply:
now i see that apart from the chord vector other two scalars are set... please rephrase the beginning of my commit suggestion 😆